### PR TITLE
New helm chart values: arguments and backtrace

### PR DESCRIPTION
**Pull request by jotak**

### CODE REVIEW

**Review by rossf7 — APPROVED** [review]
@jotak Nice 👍 thanks for adding

**Comment by bpetit** [comment]
HI ! Thanks a lot for the PR @jotak !
@rossf7 did you try it ?

File: helm/scaphandre/values.yaml [hunk 1]
```diff
@@ -13,8 +13,10 @@ resources:
 
 scaphandre:
   command: prometheus
+  args: {}
```
Comment (rossf7) [inline helm/scaphandre/values.yaml:16]:
```suggestion
  # args are added before the command
  args: {}
```

**Review by rossf7 — APPROVED** [review]
@bpetit Yes worked well in my test 👍

**Pull request merged**